app/src/main/res/values/colors.xml

```diff
@@ -76,4 +76,29 @@
   <color name="green_shade_40">#6644A85A</color>
   <color name="green_shade_10">#1A44A85A</color>
   <color name="light_green">#D9F3E7</color>
+  <!-- Avatar Background Color -->
+  <color name="avatar_background_1">#E65C5C</color>
```

Comment:
@jamesxu0 Also are these colors provided by @mschanteltc , though this is a part of lowfi, just want to confirm keeping hifi in mind.